src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -375,20 +384,6 @@ let writeReport (results: AnalyzerMessage list option) (codeRoot: string option)
         logger.LogError(ex, "Could not write sarif to {report}", report)
         logger.LogInformation("{0}", ex)
 
-let calculateExitCode (msgs: AnalyzerMessage list option) : int =
```

Comment:
This is now extracted to the bottom of the file, where the logic is inlined. That's because our exit code now depends on knowledge we only have down there, and I didn't really fancy passing more args into this function.